src/IK/src/JointLimitsTask.cpp

```diff
@@ -155,6 +155,14 @@ bool JointLimitsTask::initialize(
         return false;
     }
 
+    if (m_klim.size() != m_kinDyn->getNrOfDegreesOfFreedom())
+    {
+        log()->error("{} The size of the 'klim' parameter does not match with the number of "
```

Comment:
Could you print also here the expected/given dimensions?

Reply:
Done here: https://github.com/ami-iit/bipedal-locomotion-framework/pull/797/commits/05294ae9c67b460caa747c377078c7e6aebb1eab